src/controllers/reminder.controller.js

```diff
@@ -134,6 +137,59 @@ const reminderController = {
 		}
 	},
 
+	getOneHourToGoReminder: async (req, res) => {
```

Comment:
This Job isn't meant to be in a route normally, can it be abstracted to its own function, and this is an expensive call though.. it can be optimized.

Reply:
alright. it will be adjusted. thanks